pkg/compose/build_classic.go

```diff
@@ -65,7 +65,7 @@ func (s *composeService) doBuildClassic(ctx context.Context, project *types.Proj
 	return nameDigests, errs
 }
 
-//nolint: gocyclo
+// nolint: gocyclo
```

Comment:
It took me some time (and a ticket in the Golang issue tracker https://github.com/golang/go/issues/53835) to learn, but the correct format for these "machine-readable" comments is to not have a leading space _AND_ no space after the colon; see https://github.com/moby/moby/pull/43804

So the valid format for these (and to be left alone by `gofmt`) is;

```go
//nolint:gocyclo
```

Reply:
Ahhh thanks! I was really lazy here because I just wanted to be able to run `golanglint-ci run` from the CLI without failures/downgrading it

Did a `nolint` cleanup in #9738 😊